ddp-workspace/projects/ddp-circadia/src/app/components/activity/activity.component.ts

```diff
@@ -12,4 +14,31 @@ import {
   styleUrls: ['./activity.component.scss'],
   providers: [SubmitAnnouncementService, SubmissionManager],
 })
-export class ActivityComponent extends ActivityRedesignedComponent {}
+export class ActivityComponent extends ActivityRedesignedComponent implements OnInit {
+  ngOnInit(): void {
+    super.ngOnInit();
+
+    this.initFixedStepper();
+  }
+
+  private initFixedStepper(): void {
+    const isLoaded$ = this.getIsLoaded$();
+
+    const sub = isLoaded$.pipe(filter(isLoaded => isLoaded)).subscribe(() => {
+      if (this.model.statusCode !== ActivityStatusCodes.COMPLETE) {
+        // @ts-ignore
```

Comment:
please add a comment which explains which error is being suppressed.

Reply:
@aglushko-broad cannot say for sure is that a bug or not.
[This](https://github.com/broadinstitute/ddp-angular/blob/eaae88fad3b714d54feee259074c002fd05116f6/ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts#L504) exact line breaks the sectioning when you jump from one activity with sections to another with sections. Removing this line fixes this behavior so that on each new activity the stepper is initialized again.

This line was introduced under "Fix small problems" commit so I don't really know whether removing this line will break anything in other studies.

Reply:
As an example, if you submit an activity that has 3 sections, `currentSectionIndex` will still store `2`. Once new activity after the submission was rendered `currentSectionIndex` will not be reset to 0.

Reply:
My opinion here - it shouldn't break anything if we remove take(1), we subcribes on subject observable, doesn't look like removing take(1) will call additional server requests, also check this line
https://github.com/broadinstitute/ddp-angular/blob/develop/ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-redesigned.component.ts#L46
no take(1) and should work with no issues.

Reply:
@aglushko-broad alright, I'll remove that line in a separate PR.